src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -576,7 +583,11 @@ private BoundStatement UpdateStatement(BoundSpillSequenceBuilder builder, BoundS
             {
                 var lastSpillNode = (BoundSpillSequenceBuilder)newList[lastSpill];
                 builder.Include(lastSpillNode);
-                result.Add(lastSpillNode.Value);
+
+                var value = lastSpill > 0
```

Comment:
>lastSpill > 0 [](http://example.com/codeflow?start=28&length=13)

This condition feels arbitrary. Is it just an attempt to optimize by skipping certain shapes of the tree that we think do not exist?

Reply:
Handled differently to avoid double visit of `.Value`. 

While the intention is more clear now, this still feels a little bit out of place.. and wrong considering bootstrap build failed.

How could I reproduce the bootstrap build locally?